packages/reporting-server/README.md

```diff
@@ -65,8 +62,49 @@ postgres://<user>:<password>@<host>/<database>
 
 for more information, see https://tortoise-orm.readthedocs.io/en/latest/databases.html.
 
+## Run the application
+
+Now that you have already elected, configured and created a database and set you configurations. You need to run the migrations to create the database tables. For that we use `Aerich`:
```

Comment:
It sounds like it is not possible to run `reporting-server` without `aerich`, so instead of an extra, it should be part of the core dependencies.

Reply:
Yeah, moved to core dependencies